.github/workflows/update-docs.yml

```
@@ -0,0 +1,45 @@
+name: Update Docs
+on:
+  pull_request:
```

```suggestion
  pull_request
```
the `:` is unnecessary